### PR TITLE
fix bundle url default value

### PR DESCRIPTION
**Pull request by AndyWu2015**
as the liferay-workspace version will change and it is hard to follow the default bundle url value , we just write the current bundle url to gradle.properties and run initBundle task

### CODE REVIEW

**Comment by yanan** [comment]
Test Passed on Jenkins http://ide-build-1:9090/ on this commit 5d700f1ee22ec94c72e2ec161a6cfaaffff21243 on build 121

**Comment by gamerson** [comment]
Merged, thanks! See 734b1e0...5d700f1 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
